docs/storage.md

```diff
@@ -357,6 +357,7 @@ type: AZURE
 config:
   storage_account: ""
   storage_account_key: ""
+  storage_connection_string: ""
```

Comment:
Is this coming from the update of the library?

Reply:
Yes, it was added two weeks ago: https://github.com/thanos-io/objstore/commit/1b257a36f9a3befcea136e99c36472bc38d205de